spec/requests/admin/court_applications_controller_spec.rb

```diff
@@ -54,42 +54,46 @@
       expect(response).to be_successful
     end
 
-    it "assigns @court_application" do
-      expect(assigns(:court_application)).to eq(court_application)
+    it "loads court application details" do
+      expect(response.body).to include(court_application.id)
     end
 
-    it "assigns @defendants from prosecution_case" do
+    it "loads defendant details" do
       prosecution_case = court_application.prosecution_cases.first.reload
-      expect(assigns(:defendants)).to eq(prosecution_case.defendants)
+      prosecution_case.defendants.each do |defendant|
+        expect(response.body).to include(defendant.id).and include(defendant.defendable.person.first_name)
+      end
     end
 
-    it "assigns @hearings from prosecution_case" do
+    it "loads hearing details" do
       prosecution_case = court_application.prosecution_cases.first.reload
-      expect(assigns(:hearings)).to eq(prosecution_case.hearings)
+      prosecution_case.hearings.each do |hearing|
+        expect(response.body).to include(hearing.id)
+      end
     end
   end
 
   describe "GET #new" do
-    it "Creates court_application object" do
+    it "renders the court application form" do
```

Comment:
There is no reference to the form in this test.
Possible fix: maybe change the title (or align the test with the title)

Reply:
If you look in the views, the form template is the only place that has the `<h2>Court Application</h2>` snippet. By testing for that snippet, I am testing that the request causes the form to render.